addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -96,6 +97,7 @@ const Wysiwyg = Widget.extend({
 
         var options = this._editorOptions();
         this._value = options.value;
+        this.options.isPortalUser = await session.user_has_group('base.group_portal');
```

Comment:
Why not rather check if the user has the group which is actually allowing the ir.attachment operations?

```suggestion
        this.options.isInternalUser = await session.user_has_group('base.group_user');
```